### ISSUE TITLE
`module.exports` is not safe to read

### ISSUE THREAD

**Issue by zhaoyao91** [facebook/regenerator#406]
see https://github.com/facebook/regenerator/blob/master/packages/regenerator-runtime/runtime.js#L732

it assume `module.exports` is an object. howerver, with current webpack 5, the `module` is an object while `module.exports` is only an setter, not an object at all, which lead to the parameter to be empty.

**Comment by PatrickHollweck** [comment]
Yup, Got the same issue. Using `webpack^5` and im getting `Cannot set property 'wrap' of undefined` because `module.exports` is set only
